src/libs/actions/Task.js

```diff
@@ -743,7 +743,10 @@ function getShareDestination(reportID, reports, personalDetails) {
     const report = lodashGet(reports, `report_${reportID}`, {});
     let subtitle = '';
     if (ReportUtils.isChatReport(report) && ReportUtils.isDM(report) && ReportUtils.hasSingleParticipant(report)) {
-        subtitle = LocalePhoneNumber.formatPhoneNumber(report.participants[0]);
+        const participantAccountId = lodashGet(report, 'participantAccountIDs[0]');
+        const phoneNumber = lodashGet(personalDetails, [participantAccountId, 'phoneNumber']);
+        const login = lodashGet(personalDetails, [participantAccountId, 'login']);
+        subtitle = LocalePhoneNumber.formatPhoneNumber(phoneNumber || login);
```

Comment:
I think we should actually just use their login here and fallback on their display name if they don't have a login . And still use formatPhoneNumber so that if their login is a phone number it is properly formatted. So I think you should update line 747 to be: 

```
const displayName = lodashGet(personalDetails, [participantAccountID, 'displayName']);
```

And then update 749 to be: 

```suggestion
        subtitle = LocalePhoneNumber.formatPhoneNumber(login || displayName);
```